test/crawler_web/controllers/home_controller_test.exs

```diff
@@ -23,4 +26,72 @@ defmodule CrawlerWeb.HomeControllerTest do
       assert html_response(conn, 200) =~ "#{gettext("Log in")}</a>"
     end
   end
+
+  describe "POST /home" do
```

Comment:
prefer to create a new `keyword_controller` with a `post method` there @vnntsu

Reply:
@andyduong1920 Good point 🙏